src/transformers/modeling_albert.py

```diff
@@ -462,6 +463,9 @@ def get_input_embeddings(self):
     def set_input_embeddings(self, value):
         self.embeddings.word_embeddings = value
 
+    def get_layers(self):
```

Comment:
I like this new entry point and feel it could be useful for other new additions to the lib!